srcpkgs/avr-binutils/template

```diff
@@ -3,43 +3,25 @@ pkgname=avr-binutils
 version=2.32
 revision=1
 wrksrc="binutils-$version"
+build_style=gnu-configure
+configure_args="--disable-werror --enable-shared --enable-gold
+ --enable-plugins --enable-install-libbbfd --build=x86_64-unknown-linux-gnu
```

Comment:
This should probably read `--build=${XBPS_TRIPLET}` in case someone wants to build on a e.g. `aarch64` host.